fun_certificates/locale/fr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,70 @@
+
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: France Université Numérique 16\n"
+"Report-Msgid-Bugs-To: team@openfun.fr\n"
+"POT-Creation-Date: 2015-11-27 16:19+0100\n"
+"PO-Revision-Date: 2015-11-27 16:19+0000\n"
+"Last-Translator: FUN <team@openfun.fr>\n"
+"Language-Team: FUN <team@openfun.fr>\n"
+"Plural-Forms: nplurals=2; plural=(n > 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 1.3\n"
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:174
+msgid " It does not certify that the learner was registered with {}."
+msgstr " Elle n'atteste pas que le participant était inscrit à/au {}."
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:180
+msgid ". The learner's identity has not been verified."
+msgstr ". L’identité du participant n’a pas été vérifiée."
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:103
+msgid "ATTESTATION OF ACHIEVEMENT"
+msgstr "ATTESTATION DE SUIVI AVEC SUCCÈS"
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:162
+msgid "FUN"
+msgstr "plateforme FUN"
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:228
+msgid "MOOC: Massive Open Online Course"
+msgstr "MOOC : cours en ligne"
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:164
+#, python-format
+msgid "On the %m/%d/%Y"
+msgstr "Le %d/%m/%Y"
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:173
+msgid ""
+"The current document is not a degree or diploma and does not award "
+"credits (ECTS)."
+msgstr ""
+"La présente attestation n’est pas un diplôme et ne confère pas de crédits"
+" (ECTS)."
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:183
+msgid "The learner's identity has not been verified."
+msgstr "L’identité du participant n’a pas été vérifiée."
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:196
+msgid "Your instructors"
+msgstr "Enseignants"
+
+#: /edx/app/edxapp/fun-apps/fun_certificates/generator.py:160
+msgid "and published on "
```

Comment:
Avec https://github.com/openfun/fun-apps/pull/338#discussion_r46404847 il faudra ajouter "the": "and published on the "